src/components/Card/Card.types.ts

```diff
@@ -35,6 +33,22 @@ export interface CardHeaderProps {
    * Actions menu aria label
    */
   actionsButtonLabel?: string;
+  /**
+   * Adornment placed on the left side.
+   */
+  leftAdornment?: React.ReactNode;
+  /**
+   * Adornment placed on the right side.
+   */
+  onHelpClick?: () => void;
+  /**
+   * Title max number of lines
+   */
+  titleLinesCount?: number;
+  /**
+   * Subtitle max number of lines
+   */
+  subtitleLinesCount?: number;
```

Comment:
Let's use `maxTitleLinesCount` and `maxSubtitleLinesCount` it is more consistent wit similar cases we already have in DS